docfx_yaml/extension.py

```diff
@@ -588,7 +588,7 @@ def _update_friendly_package_name(path):
 
     # If there is no summary, add a short snippet.
     else:
-        datam['summary'] = "API documentation for `{}` {}.".format(name, _type)
+        datam['summary'] = "API documentation for `{}` {}.".format(short_name, _type)
```

Comment:
Nit: I belive `f` strings are preferred for single-line strings.

Reply:
SGTM, I'll keep this in mind for the future. There's also a lot of parts for single-lines, created #80 to follow up.